configuration/README.md

```diff
@@ -13,14 +13,26 @@ For design details concerning value and instruction classification, see [/design
 
 Configuration is provided to `go-flow-levee` via YAML.
 
-Objects of interest are identified primarily via regexp. An empty regexp will match any string.
+Sources are identified by package, type, and field names.
+You may specify these with a combination of string literals or regexp.
+Use `Package`, `Type`, and `Field` to specify by string literal.
+Use `PackageRE`, `TypeRE`, and `FieldRE` to specify by regexp.
+You may use a combination as needed.
 
-Sources are identified via regexp according to package, type, and field names.
 ```yaml
 Sources:
+# Specify using string literals
+- Package: "literal/package/path"
+  Type: "typeName"
+  Field: "fieldName"
+# Specify using regexp
 - PackageRE: "<package path regexp>"
   TypeRE: "<type name regexp>"
   FieldRE: "<field name regexp>"
+# Specify using string literals and regexp
+- Package: "literal/package/path"
+  TypeRE: ".*"
+  Field: "fieldName"
 ```
```

Comment:
This feels rather repetitive to me, I'm worried that the signal may be lost in the noise. Maybe giving just 1 concrete example and explaining it would be more helpful? E.g. you could have the type be a regex, the field be a string, and not have a package (or some other mix). Then you could have a few lines explaining what each of those mean.